doc/api/crypto.markdown

```diff
@@ -1153,6 +1168,15 @@ keys:
 
 All paddings are defined in the `constants` module.
 
+### crypto.timingSafeEqual(a, b)
+
+Returns true if a is equal to b, without leaking timing information that would
```

Comment:
Added backticks. I didn't add the SHA256 note. I feel like that's an implementation detail that should be left unspecified by the docs.